examples/guestbook/frontend-service.yaml

```diff
@@ -11,5 +11,6 @@ spec:
   ports:
     # the port that this service should serve on
     - port: 80
+      targetPort: 80
```

Comment:
This is not in one of the referenced cherry picks, nor should it be required. targetPort is, as I understand it, automatically set to the 'first' port...